modules/developer-cli-odo-preference-table.adoc

```diff
@@ -14,27 +14,24 @@ The following table shows the available options for setting preference keys for
 
 |`UpdateNotification`
 |Control whether a notification to update `odo` is shown.
-|True
-
-|`NamePrefix`
-|Set a default name prefix for an `odo` resource. For example, `component` or  `storage`.
-|Current directory name
+|`True`
 
 |`Timeout`
 |Timeout for the Kubernetes server connection check.
 |1 second
 
-|`BuildTimeout`
-|Timeout for waiting for a build of the git component to complete.
-|300 seconds
-
 |`PushTimeout`
 |Timeout for waiting for a component to start.
 |240 seconds
 
+
+|`RegistryCacheTime`
+|Duration (in minutes) `odo` will store information from the Devfile registry in the cache
```

Comment:
Does it need to be future tense? Or can we say, `odo stores information from the Defile registry in the cache.` 

If it s something odo _will_ do, then obviously keep it in future tense. It reads to me more of an explanation about what odo _does_, hence I wondered if it could be written in the present tense.

Reply:
Agreed - removed future tense